testdata/golden/TestReadSearch/ctagsrepo_v16.00000.golden

```diff
@@ -15,8 +15,11 @@
             "LineStart": 69,
             "LineEnd": 82,
             "LineNumber": 10,
+            "Before": null,
```

Comment:
based on these updates we probably need gratuitous sprinkling of `omitempty` in our api structs

Reply:
true, I will tackle this in a separate PR